resolwe_bio/processes/support_processors/report_variants.py

```diff
@@ -0,0 +1,502 @@
+        "resources": {
+            "cores": 1,
+            "memory": 8196,
+        },
+    }
+    entity = {
+        "type": "sample",
+    }
+    category = "Other"
+    data_name = 'Mutations table ({{ variants|sample_name|default("?") }})'
+    version = "1.0.0"
+    scheduling_class = SchedulingClass.BATCH
+    persistence = Persistence.CACHED
+
+    class Input:
+        """Input fields to ReportVariants."""
+
+        variants = DataField(
+            data_type="variants:vcf:snpeff",
+            label="Annotated variants",
+            description="Variants annotated with SnpEff.",
+        )
+        mutations = ListField(
+            StringField(),
+            label="Gene and its mutations",
+            description="Insert the gene you are interested in, together "
+            "with mutations. First enter the name of the gene and then "
+            "the mutations. Seperate gene from mutations with ':' and mutations "
+            "with ','. Example of an input: 'KRAS: Gly12, Gly61'. Press enter "
+            "after each input (gene + mutations). NOTE: Field only accepts "
+            "three character amino acid symbols.",
+        )
+        vcf_fields = ListField(
+            StringField(),
+            label="Select VCF fields",
+            description="The name of a standard VCF field or an "
+            "INFO field to include in the output table. "
+            "The field can be any standard VCF column (e.g. CHROM, ID, QUAL) "
+            "or any annotation name in the INFO field (e.g. AC, AF). "
+            "Required fields are CHROM, POS, ID, REF and ANN.",
+            default=[
+                "CHROM",
+                "POS",
+                "ID",
+                "QUAL",
+                "REF",
+                "ALT",
+                "DP",
+                "ANN",
+            ],
+        )
+        ann_fields = ListField(
+            StringField(),
+            label="ANN fields to use",
+            description="Only use specific fields from the SnpEff ANN "
+            "field. All available fields: Allele | Annotation | Annotation_Impact "
+            "| Gene_Name | Gene_ID | Feature_Type | Feature_ID | Transcript_BioType "
+            "| Rank | HGVS.c | HGVS.p | cDNA.pos / cDNA.length | CDS.pos / CDS.length "
+            "| AA.pos / AA.length | Distance | ERRORS / WARNINGS / INFO' ."
+            "Fields are seperated by '|'. For more information, follow this [link]"
+            "(https://pcingola.github.io/SnpEff/se_inputoutput/#ann-field-vcf-output-files).",
+            default=[
+                "Allele",
+                "Annotation",
+                "Annotation_Impact",
+                "Gene_Name",
+                "Feature_ID",
+                "HGVS.p",
+            ],
+        )
+        reference = DataField(
+            data_type="variantstable",
+            label="Table of known annotated variants",
+            description="Table of dbSNP variants annotated with SnpEff.",
```

Comment:
Expand the description to make it more clear how to prepare this reference mutation table (which process to use and how to trigger it non-default inputs are needed)